python/paddle/vision/layers.py

```diff
@@ -0,0 +1,18 @@
+#   Copyright (c) 2020 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+__all__ = ['yolo_loss', 'yolo_box']
+
+from ..fluid.layers import yolov3_loss as yolo_loss
```

Comment:
maybe add the `#DEFINE_ALIAS` annotation? 
`as yolo_box` is not needed
also, why alias `yolov3_loss` as `yolo_loss`?

Reply:
Done for `as yolo_box` and `#DEFINE_ALIAS`, `yolov3_loss` as `yolo_loss` for same naming style with `yolo_box` and `v3` should be an input attribute instead of showing in API name

Reply:
okay， I see